browser/ui/webui/settings/brave_settings_localized_strings_provider.h

```diff
@@ -15,6 +15,7 @@ class WebUIDataSource;
 namespace settings {
 
 void BraveAddLocalizedStrings(content::WebUIDataSource*, Profile*);
+void BraveAddEmailAliasesStrings(content::WebUIDataSource*);
```

Comment:
I don't think we need to add this to the header file - its only called from inside `BraveAddLocalizedStrings` so it can just live in the anonymous namespace

Reply:
fixed